recipes/rpkmforgenes/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: "rpkmforgenes"
+  version: "1.0.1"
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/danielramskold/S3_species-specific_sequencing
+  git_rev: 1.0.1
+  md5: foo
```

Comment:
md5 foo? This is quite unlikely, isn't it ;-)? We have recently changed our policy to enforce non-git urls and md5 sums for all recipes (because galaxy now provides automatic mirroring for all our sources). You can use the release tarball from github instead.